<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PwGisWidgetVersionWindow</class>
 <widget class="QWidget" name="PwGisWidgetVersionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo PwGisWidget version</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="PwGisWidget" name="pwgiswidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnOpenMap">
         <property name="text">
          <string>open map</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCloseMap">
         <property name="text">
          <string>close map</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnGetVersion">
         <property name="text">
          <string>get version</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblVersion">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
